akka-actor-typed/src/main/scala/akka/actor/typed/internal/adapter/ActorContextAdapter.scala

```diff
@@ -28,7 +28,7 @@ import akka.actor.typed.Behavior.UntypedBehavior
     ActorContextAdapter.spawnAnonymous(untyped, behavior, props)
   override def spawn[U](behavior: Behavior[U], name: String, props: Props = Props.empty) =
     ActorContextAdapter.spawn(untyped, behavior, name, props)
-  override def stop[U](child: ActorRef[U]): Boolean =
+  override def stop[U](child: ActorRef[U]): Unit =
```

Comment:
Perhaps we might still want to keep the Boolean return value to expose whether the child had already been stopped? Not sure about the use case though, why did we have that?

Reply:
Can't hink of why it was useful except for to test the internal path that was tekan.